### ISSUE TITLE
Chocobo Icons "Broken"

### ISSUE THREAD

**Issue by someONEgood** [bsides/horizoverlay#68]
Currently Chocobos are displayed in Horizoverlay without an appropriate chocobo icon; instead it defaults to the "Disconnected" icon.

Here are some replacement options (I personally prefer Icon_6):
https://ffxiv.gamerescape.com/w/images/4/4f/Player30_Icon.png
https://ffxiv.gamerescape.com/w/images/e/e4/Chocobo_Icon_7.png
https://ffxiv.gamerescape.com/w/images/a/a9/Chocobo_Icon_6.png

**Comment by bsides** [comment]
Sorry for the late reply @someONEgood! I'm back in the game and I feel the exact need.

Now the problem: ACT (or the FFXIV plugin) don't send me the data about the character race or type so I don't know what's reaching me to change it accordingly. I'm going to run some tests tonight to see if it's still the case (it's been a while) and I'll post my thoughts here then.

Thanks again for the help!

**Comment by someONEgood** [comment]
Well first of all, welcome back! I hope you enjoy yourself. If you find yourself on Crystal, let me know. It would be fun to say hi.

If you need further testing, let me know!
Also I noticed that the TRUST NPC's also show up in ACT, and lack icons, maybe that's something we can address as well. (I could potentially make icons for you).

**Comment by bsides** [comment]
Yay thanks! It's a new expansion and I'm excited 😸 

About the TRUST NPC - it's harder to test but if I can get more data about chocobos / summons / etc, I probably can get their data too. How are they showing right now? Does it show their name? If so, do you think there's a list somewhere I could borrow to manually insert into the overlay?

It goes like this: right now I get every job acronym and update [this list](https://github.com/bsides/horizoverlay/blob/aebe79f90e47c31d3ab711c9a6851bf26dbca58b/src/helpers.js#L146) manually. I could also update it with the npc's names, chocobos too - and considering ACT give me data like that, I can compare them and color accordingly.

**Comment by someONEgood** [comment]
I'm going off memory here at the moment and can't get on to double check until later this evening, but if I recall correctly they show up in Hoizoverlay (and ACT?) as (Character Name)NPC name.

So for me, (Piper Rainesong)Yshtola

Needless to say it was a bit cramped lol.

Icon was the standard "disconnected" icon.

Just wanted to get that info to you if it helps track it down.

**Comment by bsides** [comment]
I believe it does, thanks! I'll look into those this weekend for sure.

**Comment by bsides** [comment]
Yeah I can definitely get them by their names. The only problem right now is at quests and the TRUST system. In quests, some NPC seems to have the same format (Their Name (Your Name)) and the trust system also does that. For the trust system is easier because they come with job so I just rule out the ones with  it. But for quests I still don't know how to get them. Might look a little bugged though.

**Comment by bsides** [comment]
> Well first of all, welcome back! I hope you enjoy yourself. If you find yourself on Crystal, let me know. It would be fun to say hi.

Oh and about that, I'm on Aether =/
